.env.example

```
@@ -76,3 +76,4 @@ TWITCH_OAUTH_CLIENT_SECRET=
 TWITCH_OAUTH_REDIRECT_URI=
 
 DISCORD_TOKEN=
+DISCORD_GENEREAL_CHAT_ID=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)